configs/aarch64_defconfig

```diff
@@ -15,7 +15,7 @@ BR2_ROOTFS_DEVICE_CREATION_DYNAMIC_EUDEV=y
 BR2_ROOTFS_DEVICE_TABLE="system/device_table.txt ${BR2_EXTERNAL_INFIX_PATH}/board/common/xattrs"
 # BR2_TARGET_ENABLE_ROOT_LOGIN is not set
 BR2_SYSTEM_BIN_SH_BASH=y
-BR2_TARGET_GENERIC_GETTY_PORT="@console"
+BR2_TARGET_GENERIC_GETTY_PORT="console"
```

Comment:
Incoming [fix](https://github.com/kernelkit/infix/commit/e11da8a338a04663e1087a21744ca59dc22755f6) to @console, on the [console-fixes](https://github.com/kernelkit/infix/commits/console-fixes/) branch, hopefully address this issue.